decodex86/leanpkg.path

```diff
@@ -0,0 +1,2 @@
+builtin_path
+path ./src
```

Comment:
leanpkg.path is automatically generated from leanpkg.toml; I don't think we need it in the repo.